packages/connectkit/src/utils/midp.ts

```diff
@@ -0,0 +1,29 @@
+import { createStore } from 'mipd';
+
+export const createMidp = () => {
```

Comment:
```suggestion
export const createMipd = () => {
```

Reply:
lol whoops. will update. thanks ❤️

Reply:
Fixed https://github.com/family/connectkit/pull/312/commits/9e00f0d665609d239ce6d2c5cb87b94094eae07a
Good catch @jxom, thanks for the review 🙌